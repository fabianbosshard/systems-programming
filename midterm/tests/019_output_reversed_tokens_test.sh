exec "$1"  format=@c,@b,@a delim=" " <<EOF
Kiwi 3 9
Papaya 6 7
EOF
