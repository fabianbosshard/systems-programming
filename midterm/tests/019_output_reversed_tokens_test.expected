9,3,Kiwi
7,6,Papaya
